### PR TITLE
Added support for avro-util's fastserde.

### PR DESCRIPTION
**Pull request by FelixGV**
Added a "manual" optimization mode for Avro where object reuse is
exercised. The existing avro test cases are slightly refactored
but unchanged from the perspective of performance characteristics.

Fixed some of the SerFeatures characteristics for Avro.

Also tweaked the TestCaseRunner output format so that it looks a bit
more polished: added a time unit (ns) to all latency measures, made
each metric have its own line, limited latency to one digit (i.e.
100 pico granularity), added the test scenario (create, ser, deser)
next to the serde implementation name. Also added a p99 latency
metric.

### CODE REVIEW

File: tpc/src/serializers/BenchmarkBase.java [hunk 1]
```diff
@@ -25,10 +25,10 @@
     // ------------------------------------------------------------------------------------
     // Helper classes, enums
     // ------------------------------------------------------------------------------------
-    
+
```
[the remark this answers is not in the document]
Comment (FelixGV) [reply tpc/src/serializers/BenchmarkBase.java]: Ok, finally figured that out... some rogue IDE setting getting in the way. The diff should be cleaner now.

**Comment by pascaldekloe** [comment]
Thank you Felix. 🙂 The optimisations and adjustments make sense.

**Comment by FelixGV** [comment]
Wow thanks, that was fast :D !

How does the results page update work? Do you have a given hardware that you run this on and update yourself? Or do you expect the community to submit a PR to update the results page?

**Comment by pascaldekloe** [comment]
You put in effort, and thank me for accepting the fruits. 😄 I hate how people get stuck on GitHub. The owner kindly allowed me to speed things up. So yes, more community work please. I've opened issue #82 because my hardware is too old. The results page is a public wiki.

**Comment by FelixGV** [comment]
Thanks, I add a comment on the other issue.

**Pull request merged**